### PR TITLE
Reword configure dive computer

### PR DESCRIPTION
**Pull request by atdotde**
Some users understood "Configure dive computer" as the
menu entry where you configure which dive computer to
use. Reworded to make clear that this modifies the
settings on the dive computer.

Signed-off-by: Robert C. Helling <helling@atdotde.de>

### Describe the pull request:

- [ ] Bug fix
- [ ] Functional change
- [ ] New feature
- [ ] Code cleanup
- [ ] Build system change
- [ ] Documentation change
- [ ] Language translation
- [x] UI improvement

### CODE REVIEW

File: Documentation/user-manual.txt [hunk 1]
```diff
@@ -3165,7 +3165,7 @@ can be read and changed to different values. To begin, be sure the appropriate h
 is installed for the dive computer (also required for downloading dives) and the device name of the dive computer is known.
 See <<_appendix_a_operating_system_specific_information_for_importing_dive_information_from_a_dive_computer,APPENDIX A>> for information on how to do this.
 
-Once the dive computer is connected to the _Subsurface_ computer, select _File ->  Configure dive
+Once the dive computer is connected to the _Subsurface_ computer, select _File ->  Check settings on dive
```
Comment (bstoeger) [inline Documentation/user-manual.txt]: Here you write "check", below "change"...?

Comment (atdotde) [reply Documentation/user-manual.txt]: Argh!

**Comment by bstoeger** [comment]
@dirkhh, @willemferguson: comments, please? There is no code change, so from my side we can commit. But I don't feel qualified to decide on wording-issues.

**Comment by willemferguson** [comment]
No problem from my side. LGTM. Kind regards, wf

**Comment by bstoeger** [comment]
Thank you - I think we can commit.

**Pull request merged**